webpay/spa/tests/test_views.py

```diff
@@ -84,6 +84,23 @@ def test_marketplace_purchase(self, solitude):
         doc = pq(res.content)
         eq_(doc('body').attr('data-logged-in-user'), 'user@example.com')
         eq_(doc('body').attr('data-mkt-user'), 'true')
+        eq_(doc('body').attr('data-super-powers'), 'false')
+
+    @mock.patch('webpay.auth.utils.client')
+    def test_marketplace_user_can_get_super_powers(self, solitude):
+        email = 'user@example.com'
+        solitude.get_buyer.return_value = {'uuid': 'some-uuid'}
+        jwt = self.request(
+            iss='marketplace.mozilla.com', app_secret='test secret',
+            extra_req={'productData':
+                       'my_product_id=1234&buyer_email={e}'.format(e=email)})
+
+        with self.settings(USERS_WITH_SUPER_POWERS=[email]):
+            res = self.client.get('/mozpay/', {'req': jwt})
+
+        doc = pq(res.content)
+        eq_(doc('body').attr('data-logged-in-user'), email)
+        eq_(doc('body').attr('data-super-powers'), 'true')
```

Comment:
it would be cool to have tests that depend upon on seeing how the DOM changes and just API calls, but that's a minor nit.